docs/en/index.md

```diff
@@ -6,109 +6,109 @@
 
-# Publishing
+### Kubeflow Account
 
-## Build and publish an interactive dashboard
+!!! important "Attention External Users!"
+    Users external to Statistics Canada will require a cloud account granted access by the business sponsor.
 
-[![InteractiveDashboard](images/InteractiveDashboard.PNG)](/2-Publishing/R-Shiny/)
-Use **[R-Shiny](/2-Publishing/R-Shiny/)** to build interactive web apps straight
-from R. You can deploy your R-Shiny dashboard by submitting a pull request to
-our [R-Dashboards GitHub repository](https://github.com/StatCan/R-dashboards).
-![R Shiny Server](images/readme/shiny_ui.png)
+!!! note ""
+    <center>
+    [![Kubeflow is the Heart of the AAW!](./images/Kubeflow.PNG)](https://kubeflow.aaw.cloud.statcan.ca/)
+    <h3>**[👉 Click here to setup your Kubeflow account! 👈](https://kubeflow.aaw.cloud.statcan.ca/)**</h3>
+    </center>
 
-**[Dash](/2-Publishing/Dash/)** is a data visualization tool that lets you build
-an interactive GUI around your data analysis code.
+**[Kubeflow](1-Experiments/Kubeflow/)** is a powerful and flexible open source platform that has revolutionized the way we build, deploy, and manage machine learning workflows. With its robust set of tools and frameworks, Kubeflow is the perfect starting point for data scientists and developers who want to accelerate their projects and bring them to the next level. By using up Kubeflow, you can harness the full potential of cloud-native machine learning and unlock new opportunities for innovation and growth.
 
-## Explore your data
+!!! important "Attention Statistics Canada Employees!"
+    Users internal to Statistics Canada can get started right away without any additional sign up procedures, just head to  [https://kubeflow.aaw.cloud.statcan.ca/](https://kubeflow.aaw.cloud.statcan.ca/).
 
-[![Explore your data](images/ExploreData.PNG)](/2-Publishing/Datasette/) Use
-**[Datasette](/2-Publishing/Datasette/)** , an instant JSON API for your SQLite
-databases. Run SQL queries in a more interactive way!
+Kubeflow offers an unparalleled level of flexibility and customization, empowering you to create and deploy machine learning workflows that meet your specific needs and requirements. By embracing Kubeflow, you'll be at the forefront of the machine learning revolution, and who knows what incredible insights and breakthroughs you'll discover along the way.
```

Comment:
ML -> data science/data analytics

Reply:
Also noticed this entire section never actually mentioned what Kubeflow facilitates in non-abstract words: 
- User-controlled resource provisioning (CPU/.GPU/RAM/Storage)
- Notebook servers including Ubuntu noVNC, R studio, Jupyterlab(Python, R, Julia enabled), SAS